### ISSUE TITLE
At about 8930 characters JAWS won't see any more text in a textarea in Chrome

### ISSUE THREAD

**Issue by mraccess77** [FreedomScientific/VFO-standards-support#119]
### Summary

At about 8930 characters JAWS won't see any more text in a textarea in Chrome

Example:
1. Go to a text area with Chrome -- pull in several pages of text -- more than 8930 characters
2. Press control+end arrow up, left, right, etc. to read the text and hear blank.
3. Arrow to character 8929 and hear JAWS start to announce the text again

### Expected result

JAWS should announce all text in a textarea element

### Actual result
JAWS won't see any text after the 8930 character

#### JAWS version and build number
JAWS 2018.1808 
 
#### Operating System and version
Windows 7 

#### Browser and version:
Chrome 69j

**Comment by jducrot** [comment]
Cannot reproduce using JAWS 2020.2004.66 and Chrome v 81.0.4044.138
See this pen for testing: https://cdpn.io/TPG/debug/NWGBEvv
